### ISSUE TITLE
Recipe: Inserting and Querying Data

### ISSUE THREAD

**Issue by AndyGauge** [rust-lang-nursery/rust-cookbook#455]
|  |  |
| ------------- | ------------- |
| Crates  | rusqlite|
| Section  | 7.1 SQLite  |

Once the database has structure, insert and query data

**Comment by robertDurst** [comment]
I'd like to take a shot at this if it hasn't been claimed yet!

**Comment by AndyGauge** [comment]
I'm struggling with this, as this relies on #454 
You want them both?

**Comment by robertDurst** [comment]
I can, but also want to leave it open for anyone else doing Hacktoberfest :)

**Comment by AndyGauge** [comment]
That's why I made them separate, to help out with the hackathon!

**Comment by robertDurst** [comment]
Cool, can I maybe claim #454 instead then?

**Comment by lnicola** [comment]
Should we also have a recipe for transactions? Either here or separate.

**Comment by AndyGauge** [comment]
#455 claimed by @robertDurst pending #454 structure.

**Comment by robertDurst** [comment]
Cool @lnicola ! I'll keep my eyes peeled.

Thanks @AndyGauge !

**Comment by robertDurst** [comment]
Is this good to begin? @lnicola 

A bit swamped in work today but will do have this done by Thursday Night/Friday.

**Comment by lnicola** [comment]
@robertDurst You should probably base your code on #464. I'm not really planning to make any changes (unless I get some feedback otherwise), but it wasn't merged yet because of a Windows CI issue.

**Comment by lnicola** [comment]
Yeah, you'll probably want to use `rust,no_run` in your code sections to avoid that issue.

**Comment by robertDurst** [comment]
@AndyGauge and @lnicola I will have to "unclaim" this. Got very busy and will not quote have time :(

**Comment by vishalsodani** [comment]
@AndyGauge Can I claim this ?

**Comment by AndyGauge** [comment]
Yes.  Please perform Inserts and Querying in a recipe to show off Joining the two tables.  I'll create an Issue for Transactions.
